docs/approach.rst

```diff
@@ -219,7 +219,7 @@ to select the PCA components based on three widely-used model selection criterio
 * ``aic``: the Akaike Information Criterion (`AIC`_), which is the least aggressive option;
   i.e., returns the largest number of components.
 * ``kic``: the Kullback-Leibler Information Criterion (`KIC`_), which stands in the
-  middle in terms of aggressiveness.
+  middle in terms of aggressiveness. You can see how KIC related to AIC `here`_.
```

Comment:
```suggestion
  middle in terms of aggressiveness. You can see how KIC is related to AIC `here`_.
```

Reply:
My bad. Working on too many things at the same time. Thank you once again Josh!